lib/endpoints/inventory_new.dart

```diff
@@ -986,7 +986,9 @@ class InventoryV2 {
 			String itemType = item['itemType'];
 			messageBus.publish(new RequirementProgress('getItem_$itemType', email, count: count));
 			if (itemType == 'pick' || itemType == 'fancy_pick') {
-				QuestEndpoint.questLogCache[email]?.offerQuest('Q6');
```

Comment:
No, this won't be very good. Now players won't be offered the mining quest when they get a pick but instead when they get a cocktail shaker (that would be fine, but I'm not sure it's really what you intended?).

I think a much better way to go about this would be to do this:

    if (itemType == 'pick' || itemType == 'fancy_pick') {
    	//Dullite, Beryl and Sparkly
    	QuestEndpoint.questLogCache[email]?.offerQuest('Q6');
    } else if (itemType == 'cocktail_shaker') {
    	//Make Me Some Drinks
    	QuestEndpoint.questLogCache[email]?.offerQuest('Q12');
    }